### PR TITLE
remove unused

### PR DESCRIPTION
**Pull request by anandkumarpatel**
remove unused configs
- [ ] @rsandor
- [x] @und1sk0
- [x] @podviaznikov

### CODE REVIEW

File: configs/.env.production [hunk 1]
```diff
@@ -1,3 +0,0 @@
-DATADOG_HOST=localhost
-DATADOG_PORT=8125
-ROLLBAR_KEY=7b5924eccfff415d9fc00c051811e9d7
```
Comment (rsandor) [inline configs/.env.production:3]: Why aren't these needed anymore?

**Comment by rsandor** [comment]
Looks okay, am not sure we should be blowing away the DD and Rollbar stuff in `.env.production`, though.

File: configs/.env.development [hunk 2]
```diff
@@ -1,3 +0,0 @@
-DOCKER_REMOTE_API_HOST=192.168.59.103
-DOCKER_REMOTE_API_PORT=2375
-LOG_SRC=true
```
Comment (cflynn07) [inline configs/.env.development:3]: yup this can go

File: configs/.env.production (hunk 1 above)
Comment (cflynn07) [inline configs/.env.production:3]: Are they in ansible / devops-scripts?

**Comment by cflynn07** [comment]
LGTM

**Comment by forrestj** [comment]
why get rid of rollbar? also can this be merged for the delta move?

File: configs/.env.production (hunk 1 above)
Comment (anandkumarpatel) [inline configs/.env.production:3]: these are in ansible

**Comment by anandkumarpatel** [comment]
looks like these are in ansible, but not in dock-init, making change now for this

**Comment by und1sk0** [comment]
Taking you guy's word on it,

**Pull request merged**